Apps/WebClient/src/ClientApp/src/components/timeline/laboratory.vue

```diff
@@ -192,6 +192,18 @@ $radius: 15px;
                                         </div>
                                     </div>
                                 </div>
+
+                                <div class="detailSection">
```

Comment:
this only needs to show if the result is covid. Lets add a check for the entry and if is of type covid for this to show

Reply:
I could do this here but I think we need a story/task for that, mostly because the modal that says about the covid lab results doesn't handle that either!

Reply:
uhm, you are right.